### DISCUSSION TITLE
Does your server still need `Send Group`?

### DISCUSSION THREAD

**Discussion by Safihre** [sabnzbd/sabnzbd#2715] [General]
I was hoping to remove this option under Config > Servers.
If you, the user, have configured this: 
Can you check if your server also works fine if you have `Send Group` disabled in the Server's configuration?
If not: can you let us know which servers still require this?

**EDIT:** Since nobody seems to be using it, we will remove it.

**Comment by mnightingale** [comment]
I have wondered about this, I don't recall ever needing the option.

If some servers do need it could it not be made transparent to the user?

For example I don't recall the exact errors but if SAB does a body/stat etc. and gets a `4xx no newsgroup selected` response could it not then just have logic to know it needs to send group for this and all future requests to the server?

So upon every connection to the server at most one request might need to be retried.
I don't think I'd want the requirement to be persisted in the config in case the server stops needing it but it can be remember during runtime.

**Reply by Safihre** [reply]
I added the question to the code, so during Alpha/Beta/RC testing we can see if anyone pops up :)
Maybe we don't need anything at all!

**Comment by thezoggy** [comment]
iirc this is more of an artifact of the past for some really legacy usenet service providers (this was 15+ years ago) so i'd be shocked if its honestly still needed..  but i second mnight's logic that we should just be able to detect and do it if needed..  but doing it per article would prob not be the most optimal so we would just set it for the session/instance if needed or something so its just a one time hit on first article or something.  but a bit moot if its not even needed

**Reply by mnightingale** [reply]
I coudn't remember ever needing the option in the last 15 or so years but there did used to be a lot more servers or access bundled with ISP plans.

Now that there are relatively few servers since most are resellers maybe it'l lbe OK to just strip out the code for this; which I presume is more what @Safihre is interesting in rather than just the toggle.

I guess wait and see if anyone responds but I expect it can just be removed.

**Reply by Safihre** [reply]
Just to note is that I came across this because I was refactoring to get rid of the fake Duplicate priority. But that also effects the pre queue and post processing scripts, so if they need a parameter change this could be done at same time.

**Comment by zaadstra** [comment]
I have it switched on for a server, don't know why actually, the setting was copied from older programs and settings.
And it sounded like a good idea ....
I will test what happens without it.

**Reply by zaadstra** [reply]
Tested switching off the setting and everything still works ....

**Comment by puzzledsab** [comment]
Only server I've seen in the last years that required it is a free one that allowed 1 connection and extremely slow speed. I can't remember the name and I don't think it exists anymore.

**Comment by fuzzzor** [comment]
This option has been active for 15 years or more, when I first registered. It's no longer of any use to me. I'd even forgotten about it.

**Reply by thezoggy** [reply]
this is probably the case for most, hence it now has been removed.
